redis/Redis.php

```diff
@@ -2306,30 +2306,45 @@ public function brpoplpush( $srcKey, $dstKey, $timeout ) {}
     /**
      * Adds the specified member with a given score to the sorted set stored at key.
      *
-     * @param   string  $key    Required key
-     * @param   float   $score1 Required score
-     * @param   string  $value1 Required value
-     * @param   float   $score2 Optional score
-     * @param   string  $value2 Optional value
-     * @param   float   $scoreN Optional score
-     * @param   string  $valueN Optional value
+     * @param   string  $key     Required key
+     * @param   array   $options Options if needed
+     * @param   float   $score1  Required score
+     * @param   string  $value1  Required value
+     * @param   float   $score2  Optional score
+     * @param   string  $value2  Optional value
+     * @param   float   $scoreN  Optional score
+     * @param   string  $valueN  Optional value
      * @return  int     Number of values added
      * @link    https://redis.io/commands/zadd
      * @example
      * <pre>
      * <pre>
-     * $redis->zAdd('z', 1, 'v2', 2, 'v2', 3, 'v3', 4, 'v4' );  // int(2)
+     * $redis->('z', 1, 'v1', 2, 'v2', 3, 'v3', 4, 'v4' );  // int(2)
```

Comment:
`->(` is that correct syntax?

Reply:
Ahh thanks, I've fixed a typo and introduced another one.